src/core/messenger/passback.ts

```diff
@@ -205,15 +205,34 @@ const init = (register: RegisterListener): void => {
 										`Passback: ad height is ${adHeight}`,
 									);
 									void fastdom.mutate(() => {
-										const slotHeight = `${
-											adHeight + adLabelHeight
-										}px`;
+										//The 100 is to redue CLS if Teads passes back so we don't resize back to 274px for desktop if we get an mpu
+										const slotHeight =
+											getCurrentBreakpoint() === 'mobile'
+												? `${
+														adHeight + adLabelHeight
+												  }px`
+												: `${
+														adHeight +
+														adLabelHeight +
+														100
+												  }px`;
 										log(
 											'commercial',
 											`Passback: setting height of passback slot to ${slotHeight}`,
 										);
 										slotElement.style.height = slotHeight;
-
+										const passbackElement =
+											document.getElementById(
+												`${passbackElementId}`,
+											) as HTMLDivElement | null;
+										if (
+											passbackElement &&
+											getCurrentBreakpoint() ===
+												'desktop' &&
+											passbackElement.offsetHeight === 250
+										) {
+											passbackElement.style.top = '60px';
```

Comment:
I think this would be better to do with css, also as mentioned above I think the passback ad can be a different size.

Reply:
When you say the CSS you mean in DCR https://github.com/guardian/dotcom-rendering/blob/669991c0ce39b03cebb4e005a8f408ea1dfb6c9a/dotcom-rendering/src/components/ArticleContainer.tsx#L160?

I know the above link will take me to the reserved part and it will be done in step 2 https://trello.com/c/l3yqAcyP/2936-blip-reduce-layout-shift-for-inline1-teads-step-2

Reply:
Sorry, to clarify, I think checking the height of the element and adding 60px feels a little unreliable, I would suggest using flexbox to automatically centre the ad no matter what height it may be.

Reply:
I agree with you. Flexbox seems a better solution. Thank you for clarifying more on a call.